sklearn/ensemble/_hist_gradient_boosting/gradient_boosting.py

```diff
@@ -105,11 +105,10 @@ def fit(self, X, y):
         y = self._encode_y(y)
 
         # The rng state must be preserved if warm_start is True
```

Comment:
that comment is outdated. Maybe replace with e.g.:

When warm starting, we want to re-use the same seed that was used the first time fit was called (e.g. for subsampling or for the train/val split).